### ISSUE TITLE
Edit Faces is not run on gpu

### ISSUE THREAD

**Issue by 82604716** [visomaster/VisoMaster#27]
how a greate work is!!! but when Edit Faces on, cpu is speed up.

**Comment by argenspin** [comment]
Face Editor is really heavy compared to the swapper, so it might appear like its using ur CPU instead of GPU. If u are still unsure about the GPU usage, use **nvidia-smi** to check the usage instead of the Windows Task Manager, as the task manager is sometimes incorrect in reporting the GPU usage. To increase the performance while using Face Editor, go to the Settings Tab and set you Provider Priority as TensorRT or TensorRT-Engine, instead of CUDA. That should give you a decent performance boost. (It would take a while to create the Tensor cache when you use TensorRT for the first time)

**Comment by 82604716** [comment]
def lp_motion_extractor(self, img, face_editor_type='Human-Face', **kwargs) -> dict:
        kp_info = {}
        with torch.no_grad():
            # We force to use TensorRT because it doesn't work well in trt
            #if self.models_processor.provider_name == "TensorRT-Engine":

@argenspin  I fix the bug for this code，when it run in trt more than one times,it will cover the gpu momery for last, so when use it should be 
 kp_info = {
                    'pitch': preds_dict["pitch"].detach().cpu().to(self.device),
                    'yaw': preds_dict["yaw"].detach().cpu().to(self.device),
                    'roll': preds_dict["roll"].detach().cpu().to(self.device),
                    't': preds_dict["t"].detach().cpu().to(self.device),
                    'exp': preds_dict["exp"].detach().cpu().to(self.device),
                    'scale': preds_dict["scale"].detach().cpu().to(self.device),
                    'kp': preds_dict["kp"].detach().cpu().to(self.device)
                }
so,can run very well,  in trt it cost less cpu cost.
